src/scss/_profile.scss

```diff
@@ -0,0 +1,108 @@
+body,
+html {
+  height: 100%;
+  margin: 0;
+}
+
+.user-profile-picture {
+  top: 30%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  z-index: 10;
+}
+
+h1 {
+  color: var(--bs-primary);
+}
+
+.follow {
+  width: 150px;
+}
+
+p {
+  color: var(--bs-primary);
+}
+
+.baloo {
+  font-family: "baloo 2", sans-serif;
+  font-weight: 600;
+  font-size: 1.2rem;
+}
+
+.main-content {
+  flex-grow: 1;
+  border-left: 2px solid var(--bs-primary);
+  border-right: 2px solid var(--bs-primary);
+}
+
+.profile-background img {
+  width: 100%;
+  height: auto;
+}
+
+.small-user-picture {
+  width: 30px;
+  height: 30px;
+}
+
+footer p {
+  color: var(--bs-primary);
+}
+
+@media screen and (max-width: 1200px) {
```

Comment:
You should avoid using custom media queries for this assignment. Bootstrap has a suite of responsive tools that can do the vast majority of these functions without creating extra breakpoints.